### DISCUSSION TITLE
Drag window (Form) manually on Screen.

### DISCUSSION THREAD

**Discussion by Sarund9** [picoe/Eto#1948] [Q&A] [answered]
How should I go about making a Manual Window Dragging System.
A Window can be Moved by changing it's Bounds property, but using the Delta from the MouseMove event does not work properly, as the coordinates do not seem to be in scale. 
Events return PointF while the Form.Bounds is in Point (The Bounds is in Pixel Scale, I do not know what the MouseMove event is on)

**Comment by InfinityGhost** [comment, marked as answer]
Doesn't this provide that functionality? Unless this is to pass through controls for dragging, this property should suffice.

https://github.com/picoe/Eto/blob/4b4de9ba2163d128842283ca719e81f2968f6b46/src/Eto/Forms/Window.cs#L557-L564
